src/CopySatellitesIntoNetcore.targets

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="CopySatellitesIntoNetcore" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <FullFrameworkSatelliteRoot>$(BinDir)x86\Windows_NT\Debug\Output\</FullFrameworkSatelliteRoot>
+    <FullFrameworkSatelliteRoot Condition="!Exists($(FullFrameworkSatelliteRoot))">$(BinDir)Debug\Output\</FullFrameworkSatelliteRoot>
+  </PropertyGroup>
+
+  <Target Name="GenerateSatelliteItems">
+    <ItemGroup>
+        <SourceSatelliteDirectories Include="@(Languages->'$(FullFrameworkSatelliteRoot)%(Identity)\')"/>
+        <SourceSatelliteAssemblies Include="%(SourceSatelliteDirectories.Identity)$(AssemblyName).resources.dll"/>
+    </ItemGroup>
+
+    <Error
+      Condition="!Exists(%(SourceSatelliteAssemblies.Identity))"
+      Text="Can't find full framework satellite assembly (%(SourceSatelliteAssemblies.Identity)). Please do '.\cibuild --target Full --scope Compile --build-only --localized-build'"/>
+  </Target>
+
+  <Target Name="CopySatellitesIntoNetcore"
```

Comment:
No. This will all go away when we redo our build. The .net core SDK should be able to build satellites  on .net core.